labgrid/environment.py

```diff
@@ -44,10 +44,15 @@ def get_target(self, role: str = 'main') -> Target:
         return self.targets[role]
 
     def get_feature_flags(self):
-        return self.config.get_flags()
-
-    def get_target_feature_flags(self, name):
-        return self.config.get_targets().get(name).get('feature flags')
+        return self.config.get_feature_flags()
+
+    def get_target_feature_flags(self):
+        flags = []
+        for target, value in self.config.get_targets().items():
+            flags += value.get('feature flags', [])
+            if flags:
+                flags = list(set(flags))
+        return flags
```

Comment:
Why not keep the flags as a set from the beginning (and also return a set)?

Reply:
This is not yet addressed. Also, the commit message no longer matches the method names.